app/src/app/store/mapRenderSubs.ts

```diff
@@ -308,6 +316,18 @@ export const getRenderSubscriptions = (useMapStore: typeof _useMapStore) => {
       } 
     }
   );
+  
+  const filterCountiesSub = useMapStore.subscribe<[string|undefined, MapStore['getMapRef']]>(state => [state.mapOptions.currentStateFp, state.getMapRef],
+    ([stateFp, getMapRef]) => {
+      const mapRef = getMapRef()
+      if (!mapRef) return
+      const filterExpression = (stateFp ? ["==", "STATEFP", stateFp] : true) as any
+      COUNTY_LAYERS.forEach(layer => {
+        mapRef.getLayer(layer) && mapRef.setFilter(layer,  ["any", filterExpression])
+      })
```

Comment:
I: This doesn't seem to be capturing the point label id

<img width="1172" alt="image" src="https://github.com/user-attachments/assets/3b5b0cdf-6c95-401d-9209-37ee9f1ef907">

Reply:
Had a typo in the layer name but should be good now